pyvista/examples/downloads.py

```diff
@@ -7631,3 +7631,41 @@ def download_whole_body_ct_female(load=True):  # pragma: no cover
     target_file='s1380',
     load_func=_whole_body_ct_load_func,
 )
+
+
+def download_room_cff(load=True):  # pragma: no cover
+    """Download a room model in CFF format.
+
+    Parameters
+    ----------
+    load : bool, default: True
+        Load the dataset after downloading it when ``True``.  Set this
+        to ``False`` and only the filename will be returned.
+
+    Returns
+    -------
+    pyvista.MultiBlock or tuple
+        DataSet or tuple of filenames depending on ``load``.
+
+    Examples
+    --------
+    >>> import pyvista as pv
+    >>> from pyvista import examples
+    >>> dataset = examples.download_room_cff()
+    >>> pl = pv.Plotter()
+    >>> _ = pl.add_mesh(dataset)
+    >>> pl.show()
+    """
+    return _download_dataset(_dataset_room_cff, load=load)
+
+
+def _dataset_room_cff_files_func():
+    cas = _SingleFileDownloadableDatasetLoader('FLUENTCFF/room.cas.h5')
+    dat = _SingleFileDownloadableDatasetLoader('FLUENTCFF/room.dat.h5')
+    return cas, dat
+
+
+_dataset_room_cff = _MultiFileDownloadableDatasetLoader(
+    _dataset_room_cff_files_func,
+    load_func=_load_as_multiblock,
+)
```

Comment:
It is really straightforward to add a multi-file load function. Thanks @user27182 !

Reply:
You're welcome! In the future I am considering adding a public API for this, e.g. maybe a `read_all` or `read_as_multiblock` function in `fileio.py`